lib/renewals/schema.ts

```diff
@@ -0,0 +1,8 @@
+export default `
+  type Renewal {
+    id: ID!
+    usesAccessibleConvertedVan: Boolean
+    requiresWiderParkingSpace: Boolean
```

Comment:
```suggestion
    usesAccessibleConvertedVan: Boolean!
    requiresWiderParkingSpace: Boolean!
```
These should be mandatory when returned from an API